### DISCUSSION TITLE
Is it possible to open a webdav url directly

### DISCUSSION THREAD

**Discussion by jballment** [iterate-ch/cyberduck#15575] [Q&A]
Hi,

I've been using Mountain Duck for a while to connect to our internal nextcloud server.  Every now and again someone sends me an internal URl for a file i.e. https://nextcloud/index.php/f/1123454.  I already have mountain duck set up to connect to nextcloud but is there anyway that I can use that url to open the file via mountain duck rather than having to download or open the link and find to file location and then manually navigate to the file?

Regards

John.

**Comment by budachst** [comment]
Probably not, since MD cannot know about the location of that file. I don't think, that these links are served via WebDAV, but via regular http. To check that try to open such a link via the Finder's "Connect to server" option, I doubt, that this will work.
